static/app/components/forms/formField/index.tsx

```diff
@@ -406,24 +404,31 @@ class FormField extends React.Component<FormFieldProps> {
                 }
 
                 return (
-                  <PanelAlert type={saveMessageAlertType}>
-                    <MessageAndActions>
-                      <div>
-                        {typeof saveMessage === 'function'
-                          ? saveMessage({...props, value})
-                          : saveMessage}
-                      </div>
-                      <ButtonBar gap={1}>
-                        <Button onClick={this.handleCancelField}>{t('Cancel')}</Button>
+                  <PanelAlert
+                    type={saveMessageAlertType}
+                    trailingItems={
+                      <React.Fragment>
+                        <Button
+                          onClick={this.handleCancelField}
+                          size="xsmall"
+                          key="cancel-button"
+                        >
+                          {t('Cancel')}
+                        </Button>
                         <Button
                           priority="primary"
-                          type="button"
+                          size="xsmall"
                           onClick={this.handleSaveField}
+                          key="save-button"
```

Comment:
do you need these keys?

Reply:
oh true, no need for these. they're from before when I put in an array instead of a `Fragment`.